astropy/time/core.py

```diff
@@ -864,7 +864,9 @@ def _apply(self, method, *args, **kwargs):
             new_format = self.format
 
         if callable(method):
-            apply_method = lambda array: method(array, *args, **kwargs)
```

Comment:
I personally think that `lambda` is a perfectly good way to code this particular expression (I know there are differences of opinion).  But in any case isn't this out of scope for this PR which is about `super()`?

Reply:
It seems more like a PEP8 change, which one might as well do, but I don't want to let this detract from the main goal, so undid the change.